Modules/_struct.c

```diff
@@ -497,9 +497,7 @@ nu_ulonglong(_structmodulestate *state, const char *p, const formatdef *f)
 static PyObject *
 nu_bool(_structmodulestate *state, const char *p, const formatdef *f)
 {
-    _Bool x;
-    memcpy(&x, p, sizeof x);
-    return PyBool_FromLong(x != 0);
+    return PyBool_FromLong((*p) != 0);
```

Comment:
If we want to handle the cases where `_Bool` is not one byte, I think we can do something like the following:

```c
_Bool zero = 0;
return PyBool_FromLong(memcmp(p, &zero, sizeof(_Bool)) != 0);
```

Otherwise, we probably want the `static_assert` like @alex suggested.

Reply:
I swear I didn't see this before I posted it! :)